client/blocks/importer/wordpress/import-everything/pre-migration/credentials-form.tsx

```diff
@@ -28,14 +29,20 @@ interface Props {
 	selectedHost: string;
 	migrationTrackingProps: StartImportTrackingProps;
 	onChangeProtocol: ( protocol: CredentialsProtocol ) => void;
+	allowFtp?: boolean;
 }
 
 export const CredentialsForm: React.FunctionComponent< Props > = ( props ) => {
-	const { sourceSite, targetSite, migrationTrackingProps, startImport } = props;
+	const { sourceSite, targetSite, migrationTrackingProps, startImport, allowFtp } = props;
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	const { hostname } = new URL( sourceSite.URL );
-	const [ formState, setFormState ] = useState( { ...INITIAL_FORM_STATE, host: hostname } );
+	const [ formState, setFormState ] = useState( {
+		...INITIAL_FORM_STATE,
+		host: hostname,
+		protocol: 'ssh',
+		port: 22,
```

Comment:
This configuration is directly related to the `allowFtp` property.
Shall we add this config only when `allowFtp` is `true`?

Reply:
I went ahead and did a little more enhancement here: https://github.com/Automattic/wp-calypso/pull/85818/commits/c7ed495f34f60e9da8d994782ad0495d19c1b718